alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -512,6 +517,38 @@ private function getConfigByVersion($version){
 		return array($config,$productVersion);
 	}
 
+	private function maybeAddAnalyticsPlugins($confVarsArr)
+	{
+		$ovpPlayerConfig = isset($this->bundleConfig[self::KALTURA_OVP_PLAYER]) ? $this->bundleConfig[self::KALTURA_OVP_PLAYER] : "";
```

Comment:
Use '' instead "", since double brackets requires processing of potential vars , while single quote is just a string

Reply:
Thanks @MosheMaorKaltura. fixed.